test/GameLogic/gameLogic.test.js

```diff
@@ -76,5 +76,17 @@ describe('GameLogic', () => {
             const winDetected = game.containsWinningColumn();
             expect(winDetected).toEqual(false);
         });
+
+        test('will return false boolean value when a column filled with the same token is not detected', () => {
```

Comment:
can you rename this test to "will return false boolean when a column is not filled with the same token"?